events/events.go

```diff
@@ -139,29 +159,34 @@ func StartMetricSending(nodeName string, latitude float64, longitude float64, c
 				log.LogNoRequestID("catabalancer failed to get mist state", "err", err)
 				continue
 			}
-			for stream := range mistState.ActiveStreams {
-				streamsEvent := NodeStreamsEvent{
-					Resource: nodeStreamsEventResource,
-					NodeID:   nodeName,
-					Stream:   stream,
-					IsIngest: mistState.IsIngestStream(stream),
-				}
-				payload, err := json.Marshal(streamsEvent)
-				if err != nil {
-					log.LogNoRequestID("catabalancer failed to marhsal node stats", "err", err)
-					continue
-				}
 
-				err = c.BroadcastEvent(serf.UserEvent{
-					Name:    "node-streams",
-					Payload: payload,
-				})
-				if err != nil {
-					log.LogNoRequestID("catabalancer failed to send streams info", "err", err)
-					continue
+			streamsEvent := NodeStreamsEvent{
+				Resource: nodeStreamsEventResource,
+				NodeID:   nodeName,
+			}
+			var nonIngestStreams, ingestStreams []string
+			for streamID := range mistState.ActiveStreams {
+				if mistState.IsIngestStream(streamID) {
+					ingestStreams = append(ingestStreams, streamID)
 				}
```

Comment:
Are you missing adding to `nonIngestStreams`?

Reply:
oof great catch, fixed